### DISCUSSION TITLE
csv export doesn't work with UK demographics example

### DISCUSSION THREAD

**Discussion by mikad98** [synthetichealth/synthea#1130] [Q&A] [answered]
I managed to get the "Other Areas" example running with the respective UK files. However, if I want to export data in the csv-format (exporter.csv.export = true), I get the following error:

java.lang.ExceptionInInitializerError
        at org.mitre.synthea.export.CSVExporter.getInstance(CSVExporter.java:374)
...
Caused by: java.lang.IllegalArgumentException: Null charset name
        at java.base/java.nio.charset.Charset.lookup(Charset.java:454)
...

Do you have any idea how to solve this? Thank you so much for your help!

**Comment by jawalonoski** [comment, marked as answer]
Your `synthea.properties` file needs to have a charset specified in `exporter.encoding`.

Not sure when this was added, but apparently it did not come with a default value in the code.

For example,

```properties
exporter.encoding = UTF-8
```

I'm going to create an issue, because this is a bug.

**Reply by jawalonoski** [reply]
Created #1131

**Reply by jawalonoski** [reply]
This should be fixed now.
